corehq/apps/userreports/const.py

```diff
@@ -16,6 +16,12 @@
     'In order to view this data using the Report Builder you will have to delete this report '
     'and then build it again. Click below to delete it.'
 )
+FORM_NOT_FOUND_ERROR_MESSAGE = _(
+    'Sorry! There was a problem trying to edit your report. '
+    'This likely occurred because form associated with this report was deleted from the application. '
```

Comment:
```suggestion
    'This likely occurred because a form associated with this report was deleted from the application. '
```
"a" or "the"? It is possible a specific form from an app was deleted and causes this issue right?

Reply:
the... whoops. fixed